test/acceptance/request_check.test.js

```diff
@@ -6,163 +6,126 @@
-import StatusPOM from './PageObjectModels/statusPOM'
-import ResultsPOM from './PageObjectModels/resultsPOM'
-
-test.describe('initial test container tests', async () => {
-  let container
-
-  test.beforeAll(async () => {
-    container = await new LocalstackContainer('localstack/localstack:latest').start()
-  })
-
-  test.afterAll(async () => {
-    await container.stop()
-  })
-
-  test.skip('should create a S3 bucket', async () => {
-    const client = new S3Client({
-      endpoint: container.getConnectionUri(),
-      forcePathStyle: true,
-      region: 'us-east-1',
-      credentials: {
-        secretAccessKey: 'test',
-        accessKeyId: 'test'
-      }
-    })
-    const input = {
-      Bucket: 'testcontainers'
-    }
-    const command = new CreateBucketCommand(input)
-
-    const createBucketResponse = await client.send(command)
-    expect(createBucketResponse.$metadata.httpStatusCode).toEqual(200)
-    const headBucketResponse = await client.send(new HeadBucketCommand(input))
-    expect(headBucketResponse.$metadata.httpStatusCode).toEqual(200)
+import { test } from '@playwright/test'
 
-    await container.stop()
-  })
-})
+import StartPage from './PageObjectModels/startPage'
+import DatasetPage from './PageObjectModels/datasetPage'
+import UploadMethodPage from './PageObjectModels/uploadMethodPage'
+import UploadFilePage from './PageObjectModels/uploadFilePage'
+import StatusPage from './PageObjectModels/statusPage'
+import ResultsPage from './PageObjectModels/resultsPage'
 
 test.describe('Request Check', () => {
   test.describe('with javascript enabled', () => {
     test.skip('request check of a datafile', async ({ page }) => {
-      const startPOM = new StartPOM(page)
-      const datasetPOM = new DatasetPOM(page)
-      const uploadMethodPOM = new UploadMethodPOM(page)
-      const uploadFilePOM = new UploadFilePOM(page)
-      const statusPOM = new StatusPOM(page)
-      const resultsPOM = new ResultsPOM(page)
-
-      await startPOM.navigateHere()
-      await startPOM.clickStartNow()
-
-      await datasetPOM.waitForPage()
-      await datasetPOM.selectDataset(DatasetPOM.datasets.Article_4_direction_area_dataset)
-      await datasetPOM.clickContinue()
-
-      await uploadMethodPOM.waitForPage()
-      await uploadMethodPOM.selectUploadMethod(UploadMethodPOM.uploadMethods.File)
-      await uploadMethodPOM.clickContinue()
-
-      await uploadFilePOM.waitForPage()
-      await uploadFilePOM.uploadFile('test/datafiles/Article_4_direction_area_dataset.csv')
-      await uploadFilePOM.clickContinue()
-
-      await statusPOM.waitForPage()
-      await statusPOM.expectStatusToBe('NEW')
-      await statusPOM.expectStatusToBe('IN_PROGRESS')
-      await statusPOM.waitForContinueButton()
-      await statusPOM.expectStatusToBe('COMPLETE')
-      await statusPOM.clickContinue()
-
-      await resultsPOM.waitForPage()
-      await resultsPOM.expectPageIsNoErrorsPage()
+      const startPage = new StartPage(page)
```

Comment:
As discussed it would be cleaner to have the "action" methods of each page return the resultant page, i.e. when `startPage.clickStartNow()` is called, it returns `DatasetPage`.  Then the code can take a more readable form:

```
const startPage = new StartPage(page)
const datasetPage = await startPage.clickStartNow()
const uploadMethodPage = datasetPage.submitDataset(bla)
```

Note that the functionality behind `waitForPage` methods (i.e. page.waitForURL(`**${this.url}`)) can be moved into the respective page object constructor methods.  They then do not need to be called explicitly.